.github/workflows/ci.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
         make ci
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: shogo82148/actions-goveralls@v1
+      with:
+        path-to-profile: coverage.out
     - uses: actions/upload-artifact@v2
```

Comment:
Can we delete this now?

Reply:
Done, although I'm not 100% sure what the purpose of that was but it seems to try to a similar thing given that it has `coverage` in it.